build.properties

```diff
@@ -38,6 +38,12 @@
 # will cause the suffix to become blank.
 jfx.release.suffix=-ea
 
+# The experimental release suffix is set only in jfx-sandbox repository
+# for a speficic branch. When this property is set it overrides
```

Comment:
Do you want to say something about it needing to start with a `-`? Or do you think the example of `-ea` above will be clear enough?

Reply:
Added a comment to describe the use of -, with example of -metal and -d3d12.